Misc/ACKS

```diff
@@ -495,6 +495,7 @@ Lele Gaifax
 Santiago Gala
 Yitzchak Gale
 Matthew Gallagher
+Florent Gallaire
```

Comment:
Florent definitely belongs in ACKS.  We have been fairly liberal in adding people, some for as little as a well-crafted sentence or line of code.